shared/schemas.ts

```diff
@@ -11,3 +11,9 @@ export const updateSchema = z.object({
   identifier: participantIdentifierSchema,
   newIdentifier: participantIdentifierSchema,
 });
+
+export const interviewIdSchema = z.object({
+  id: z.string(),
```

Comment:
I'm not sure this is worth a shared schema. Can't we infer from the prisma model?

Reply:
resolved [400e1ec](https://github.com/complexdatacollective/Fresco/pull/20/commits/400e1ec3c3aa14a9f73e75a234a2aa3c9cb4fd94). 

Decided to move schema directly into input because deriving zod schema from existing types is not yet supported.